SeaPublicWebsite/appsettings.Production.json

```diff
@@ -7,5 +7,9 @@
                 "Args": { "uri": "tls://34ae15fb-5760-47f5-a1b8-93568b5df4d3-ls.logit.io:17202" }
             }
         ]
+    },
+
+    "GovUkNotify": {
+        "BaseUrl": "https://sea-beta-production.cloudapps.digital/"

```

Comment:
This doesn't have /energy-efficiency which seems correct, but the one below has

Reply:
Thanks, that's what I get for rushing